docs/blog/2024-10-03-nx-20-release.md

```diff
@@ -6,7 +6,15 @@ tags: [nx, release]
 cover_image: /blog/images/2024-10-03/nx-20-header.png
 ---
 
-I know it's hard to believe but Nx 20 is here! There's a lot to cover in this release, so let's just dive right in.
+I know it's hard to believe but Nx 20 is here! There's a lot of great updates in this release, but look back at some of the major features from Nx 19:
+
+- Added [`nx import`](nx-import) to bring import existing projects while maintaining git history.
```

Comment:
where should all these links point to? like `nx-import`, `crystalize-command` etc? they don't work right now.